.gitignore

```diff
@@ -20,6 +25,14 @@ bld/
 [Bb]in/
 [Oo]bj/
 
+# CMake
+CMakeCache.txt
+CMakeFiles*
```

Comment:
aargh. ignore CMakeFiles.txt ?

I think you're trying to ignore files because you did an "in-source" build, which isn't recommended in any case.